org_fedora_oscap/content_handling.py

```diff
@@ -29,6 +29,7 @@
 from collections import namedtuple, OrderedDict
 from openscap_api import OSCAP
 from pyanaconda.iutil import execReadlines
+from HTMLParser import HTMLParser
```

Comment:
Could you please prepare for python3 here? Should be a very easy change. Try to import the new name, if it fails import the old name. It's a fairly standard pattern.

See https://docs.python.org/2/library/htmlparser.html